web/src/common/CountDownInput.js

```diff
@@ -79,13 +82,14 @@ export const CountDownInput = (props) => {
         setCaptchaImg(res.captchaImage);
         setCheckType("Default");
         setVisible(true);
-      } else if (res.type === "reCAPTCHA" || res.type === "hCaptcha") {
+      } else {
         setCheckType(res.type);
         setClientId(res.clientId);
         setCheckId(res.clientSecret);
         setVisible(true);
-      } else {
-        Setting.showMessage("error", i18next.t("signup:Unknown Check Type"));
```

Comment:
if `Setting` is not used, plz remove import codes `import * as Setting from "../Setting";` to fix the warning

![image](https://user-images.githubusercontent.com/33992371/176343949-8ed5cf01-c8f5-4a2b-8f9a-2211b0670e39.png)